packages/celo-eslint/index.js

```diff
@@ -0,0 +1,63 @@
+const path = require('path');
+
+module.exports = {
+  env: {
+    browser: true,
+    es2021: true,
+    node: true,
+    jest: true,
+  },
+  extends: [
+    'eslint:recommended',
+    'plugin:react/recommended',
+    'plugin:react-hooks/recommended',
```

Comment:
many project wont need react linters

Reply:
It won't cause errors in projects that don't use react, but are you thinking it should just be removed completely?

Reply:
i dont know, maybe its fine.